.rubocop.yml

```diff
@@ -1,20 +1,44 @@
 Rails/CreateTableWithTimestamps:
   Enabled: false
 Rails/HasManyOrHasOneDependent:
-  Enabled: false
+  Enabled: true
+# if a codebase is on an older version on rails override this in the local rubocop.yml
 Rails/HttpPositionalArguments:
-  Enabled: false
+  Enabled: true
 Rails/LexicallyScopedActionFilter:
   Enabled: false
 Rails/Presence:
-  Enabled: false
+  Enabled: true
 Rails/FilePath:
   EnforcedStyle: arguments
-# We should enable this later but there is currently a bug in rubocop which
-# breaks this
-# Rails/RequestReferer:
-#   EnforcedStyle: referrer
-#
+Rails/RequestReferer:
+  EnforcedStyle: referrer
```

Comment:
I did not encounter a problem enabling this, curious what the previous bug documented was

Reply:
it probably was released recently. as of the comment it was fixed in main but not released as a new gem version yet